File: 01-02-2024/maioridade/maioridade.c
#include <stdio.h>

int main() {
    int age;

    printf("Insira sua idade: ");
    scanf("%d", &age);

    if (age >= 18) {
        printf("Você é maior de idade.");
    } else {
        printf("Você é menor de idade.");
    }

    printf("\n");

    return 0;
}

File: 01-02-2024/maioridade/Maioridade.java
import java.util.Scanner;

public class Maioridade {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.println("Insira sua idade: ");

            final int age = scanner.nextInt();
            final String state;

            if (age >= 18) {
                state = "maior";
            } else {
                state = "menor";
            }

            System.out.format("Você é %s de idade.", state).println();
        } catch (Exception exception) {
            exception.printStackTrace();
        }
    }

}